src/main.rs

```diff
@@ -54,7 +64,12 @@ fn main() {
 
     let handle = thread::spawn(move || -> Result<Vec<Record>, LustreCollectorError> {
         let lctl_output = get_lctl_output()?;
-        let lctl_record = parse_lctl_output(&lctl_output)?;
+        let lctl_name_only_output = get_lctl_name_only_output()?;
```

Comment:
I'm not convinced having a separate parser just for the name only is the best way to handle this.

Reply:
This has been updated to instead use a new module and pull this information using the modules parser in its own thread.